addon-test-support/@ember/test-helpers/settled.js

```diff
@@ -198,10 +198,8 @@ export function isSettled() {
   @public
   @returns {Promise<void>} resolves when settled
 */
-export default function settled() {
-  let options = arguments[0];
-
-  if (arguments.length > 0) {
+export default function settled(options) {
```

Comment:
this was `documentation` will see that there is a parameter and document it as such, but as mentioned in https://github.com/emberjs/ember-test-helpers/issues/314 this parameter is kinda private so we should keep `let options = arguments[0];` instead.

Reply:
@Turbo87 Another good point.  How about now 😆 

https://github.com/emberjs/ember-test-helpers/pull/303/commits/22d1538396d5bcbb1ddb685d969179fc05fd01c9